src/main/java/com/crud/tasks/domain/Trello.java

```diff
@@ -0,0 +1,19 @@
+package com.crud.tasks.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Zastanowiłbym się, czy tutaj nie "jedziesz na autopilocie" - czy na pewno potrzebujesz obu konstruktorów? Sprawdzałaś to?